components/automate-chef-io/content/docs/on-prem-builder.md

```diff
@@ -231,6 +231,17 @@ Select your Gravatar icon on the top right corner of the Chef Habitat Builder we
 
 Once you are signed in to the Chef Habitat Builder UI, select the **New Origin** button and enter `core` as the name for the origin.
 
+## Access Chef Habitat Builder On-prem With Habitat Command-Line Tools
+
+Use the URL `https://{{< example_fqdn "automate" >}}/bldr/v1` (note the `v1`) when
```

Comment:
@yzl Is this URL technically always the Builder API endpoint of the installation? Asking since you're referencing this section with the link text "Builder API endpoint of your installation" in line 343.

Reply:
@mjingle If someone is using on-prem Builder deployed with Automate, then the Builder API endpoint is `https://{{< example_fqdn "automate" >}}/bldr/v1`.

If someone is using pre-Automate on-prem Builder or public Builder, then the endpoint is something like `https://example.com/bldr` without the v1.